app/views/include/forms2/errorSummary.scala.html

```diff
@@ -16,18 +16,22 @@
 
 @import forms._
 
-@(form: Form2[_])(implicit lang:Lang,  m: Messages)
+@(form: Form2[_], location: String = "")(implicit lang:Lang,  m: Messages)
 
 @if(form.errors.nonEmpty) {
     <div class="flash amls-error-summary" role="alert"  tabindex="-1">
         <h2 class="heading-medium" id="error-summary-heading">@Messages("err.summary")</h2>
         <ul class="error-summary-list">
             @form.errors.map { case (path, errors) =>
                 <li role="tooltip" class="validation-summary-message">
-                    @if(path.path.nonEmpty) {
-                        <a href="#@form(path).id">@errors.toMessage</a>
+                    @if(location.nonEmpty) {
```

Comment:
Set @link variable possibly? And then have `<a href="#@link">@errors.toMessage</a>`